docs/developer-guide/code-conventions.md

```diff
@@ -36,19 +36,7 @@ module.exports = {
 };
 ```
 
-The `@author` field gives you credit for having created the file. The `@copyright` field indicates that you are the copyright holder for the file.
-
-Your submission may touch other parts of the ESLint code that you did not write. In that case, you are welcome to add a copyright notice to the top of the file if you have done any amount of significant work on the file (we leave it up to you to decide what "significant" means - if you aren't sure, just ask). You should never change the `@author` field, but you can add another `@copyright` field on top of the existing ones, such as:
-
-```js
-/**
- * @fileoverview Description of the file
- * @author Author's Name
- * @copyright 2015 Your Name. All rights reserved.
- * @copyright 2014 Author's Name. All rights reserved.
- * See LICENSE file in root directory for full license.
- */
-```
+The `@author` field gives you credit for having created the file. There is no need to assert copyright on individual files.
```

Comment:
Could use some help here on whether I should mention the jQuery license, or just remove the sentence about copyright altogether, or something else.